src/core_ocean/shared/mpas_ocn_vel_forcing_rayleigh.F

```diff
@@ -34,15 +34,16 @@ module ocn_vel_forcing_rayleigh
    ! Public parameters
    !
    !--------------------------------------------------------------------
+   real (kind=RKIND), public :: rayleighDampingCoef=0.0_RKIND, &
+                                rayleighBottomDampingCoef=0.0_RKIND
 
    !--------------------------------------------------------------------
    !
    ! Public member functions
    !
    !--------------------------------------------------------------------
 
-   public :: ocn_vel_forcing_rayleigh_tend, &
-             ocn_vel_forcing_rayleigh_init
+   public :: ocn_vel_forcing_rayleigh_init
```

Comment:
I think it makes more sense to remove this module completely, rather than have a module file just for an init routine to obtain two variables. Could you just delete it, and just grab `rayleighDampingCoef,  rayleighBottomDampingCoef` in the vmix init routine? I can do it if you don't have time.

Reply:
Agreed, this makes the PR much cleaner and simplifies the code base.